<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-03T11:35:46 IST" name="Suite1234" finished-at="2022-06-03T11:37:17 IST" duration-ms="90516">
    <groups>
    </groups>
    <test started-at="2022-06-03T11:35:46 IST" name="ChromeTest" finished-at="2022-06-03T11:36:30 IST" duration-ms="43824">
      <class name="pack.Testng">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:35:46 IST" name="launchBrowser" finished-at="2022-06-03T11:35:49 IST" duration-ms="2118" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="pomClassObject()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:35:49 IST" name="pomClassObject" finished-at="2022-06-03T11:35:49 IST" duration-ms="46" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pomClassObject -->
        <test-method is-config="true" signature="loginToApplication()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:35:49 IST" name="loginToApplication" finished-at="2022-06-03T11:35:56 IST" duration-ms="7446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method signature="aerifyHomeTab()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:35:56 IST" name="aerifyHomeTab" finished-at="2022-06-03T11:36:04 IST" duration-ms="7459" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [(5) Facebook] but found [(3) Facebook]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [(5) Facebook] but found [(3) Facebook]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pack.Testng.aerifyHomeTab(Testng.java:93)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aerifyHomeTab -->
        <test-method is-config="true" signature="logoutFromApplication(org.testng.ITestResult)[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:04 IST" name="logoutFromApplication" finished-at="2022-06-03T11:36:08 IST" duration-ms="4249" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=aerifyHomeTab status=FAILURE method=Testng.aerifyHomeTab()[pri:0, instance:pack.Testng@6a400542] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="loginToApplication()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:08 IST" name="loginToApplication" finished-at="2022-06-03T11:36:15 IST" duration-ms="6823" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method signature="berifyfriendTab()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:15 IST" name="berifyfriendTab" finished-at="2022-06-03T11:36:20 IST" duration-ms="5105" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.facebook.com/?sk=welcome] but found [https://www.facebook.com/friends]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.facebook.com/?sk=welcome] but found [https://www.facebook.com/friends]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pack.Testng.berifyfriendTab(Testng.java:127)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- berifyfriendTab -->
        <test-method is-config="true" signature="logoutFromApplication(org.testng.ITestResult)[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:20 IST" name="logoutFromApplication" finished-at="2022-06-03T11:36:24 IST" duration-ms="4257" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=berifyfriendTab status=FAILURE method=Testng.berifyfriendTab()[pri:0, instance:pack.Testng@6a400542] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="clearBrowser()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:24 IST" name="clearBrowser" finished-at="2022-06-03T11:36:26 IST" duration-ms="2008" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearBrowser -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:pack.Testng@6a400542]" started-at="2022-06-03T11:36:26 IST" name="closeBrowser" finished-at="2022-06-03T11:36:30 IST" duration-ms="4266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- pack.Testng -->
    </test> <!-- ChromeTest -->
    <test started-at="2022-06-03T11:36:30 IST" name="EdgeTest" finished-at="2022-06-03T11:37:17 IST" duration-ms="46627">
      <class name="pack.Testng">
        <test-method is-config="true" signature="launchBrowser(java.lang.String)[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:30 IST" name="launchBrowser" finished-at="2022-06-03T11:36:34 IST" duration-ms="3308" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Edge]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method is-config="true" signature="pomClassObject()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:34 IST" name="pomClassObject" finished-at="2022-06-03T11:36:34 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pomClassObject -->
        <test-method is-config="true" signature="loginToApplication()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:34 IST" name="loginToApplication" finished-at="2022-06-03T11:36:41 IST" duration-ms="7791" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method signature="aerifyHomeTab()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:41 IST" name="aerifyHomeTab" finished-at="2022-06-03T11:36:49 IST" duration-ms="7315" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [(5) Facebook] but found [(3) Facebook]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [(5) Facebook] but found [(3) Facebook]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pack.Testng.aerifyHomeTab(Testng.java:93)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aerifyHomeTab -->
        <test-method is-config="true" signature="logoutFromApplication(org.testng.ITestResult)[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:49 IST" name="logoutFromApplication" finished-at="2022-06-03T11:36:53 IST" duration-ms="4188" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=aerifyHomeTab status=FAILURE method=Testng.aerifyHomeTab()[pri:0, instance:pack.Testng@479d31f3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="loginToApplication()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:36:53 IST" name="loginToApplication" finished-at="2022-06-03T11:37:00 IST" duration-ms="7269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method signature="berifyfriendTab()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:37:00 IST" name="berifyfriendTab" finished-at="2022-06-03T11:37:06 IST" duration-ms="5927" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.facebook.com/?sk=welcome] but found [https://www.facebook.com/friends]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.facebook.com/?sk=welcome] but found [https://www.facebook.com/friends]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pack.Testng.berifyfriendTab(Testng.java:127)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- berifyfriendTab -->
        <test-method is-config="true" signature="logoutFromApplication(org.testng.ITestResult)[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:37:06 IST" name="logoutFromApplication" finished-at="2022-06-03T11:37:11 IST" duration-ms="4482" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=berifyfriendTab status=FAILURE method=Testng.berifyfriendTab()[pri:0, instance:pack.Testng@479d31f3] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method is-config="true" signature="clearBrowser()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:37:11 IST" name="clearBrowser" finished-at="2022-06-03T11:37:13 IST" duration-ms="2002" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearBrowser -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:pack.Testng@479d31f3]" started-at="2022-06-03T11:37:13 IST" name="closeBrowser" finished-at="2022-06-03T11:37:17 IST" duration-ms="4329" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- pack.Testng -->
    </test> <!-- EdgeTest -->
  </suite> <!-- Suite1234 -->
</testng-results>
